src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
@doudou, I've already pushed it  I don't know the whole Vizkit integration mechanism so it is important to see if what I did can impact somehow the other functionalities, for instance, ignoring important information for the other parts of the code. Locally, as far as I could see everything is running ok.